Direct lambda invocation add new span type (#226)
instana/go-sensor: instrumentation/instaawssdk/README.md

}
```

### Instrumenting lambda

If a session is instrumented, it will propagate tracing context automatically using values from the `ctx`.

Example:
```go
sensor := instana.NewSensor("my-new-sensor")
sess, _ := session.NewSession()
instaawssdk.InstrumentSession(sess, sensor)
svc := sdk.New(sess)
input := &sdk.InvokeInput{
    FunctionName: "my-lambda-function-name",
    // this field is optional
    // IMPORTANT type `Event` is not supported by the instrumentation
    InvocationType: aws.String("RequestResponse"), 
    Payload: []byte("{}"),
}

// invoke with context, otherwise, you will need to set context manually to propagate tracing data
svc.InvokeWithContext(ctx, input)
```

Tracing context propagated inside a `ClientContext.Custom` field in the `InvokeInput` object. Reserved keys are:
- `X-INSTANA-T`
- `X-INSTANA-S`
- `X-INSTANA-L`

> - `TRACEPARENT`

Author: I will remove support for w3c context

- `TRACESTATE`

> To avoid a collision, do not set them in your application code.

Reviewer: ```suggestion
To avoid collisions, it is recommended to avoid setting them in your application code.
```

> Known limitation:

Reviewer: ```suggestion
Known limitations:
```
